tools/android/MASTG-TOOL-0111.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Blutter
+platform: android
+source: https://github.com/worawit/blutter
+---
+
+[Blutter](https://github.com/worawit/blutter) is an open-source tool created to support the reverse engineering of Flutter applications by compiling the Dart AOT Runtime. It targets the lib files found in decompiled apks and is compatible with the latest versions of Dart. It makes use of an advanced C++20 formatting library. The tool is compatible with Linux, Windows, and macOS operating systems. Blutter is capable of extracting and analyzing Dart objects and can generate Frida scripts for further analysis. It automatically compiles any required Dart versions that are not already installed.
```

Comment:
```suggestion
[Blutter](https://github.com/worawit/blutter) is an open-source tool created to support the reverse engineering of Flutter applications. Unlike other Flutter tools, Blutter parses the libapp.so file locally, without requiring you to run the app on a device. The tool is compatible Linux, Windows, and macOS, but can only analyse Android ARM64 apps. Blutter is capable of extracting and analyzing Dart objects and it can generate Frida scripts for further analysis.
```